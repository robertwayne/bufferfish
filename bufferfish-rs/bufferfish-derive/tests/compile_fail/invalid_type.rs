use bufferfish_derive::Serialize;

#[derive(Serialize, Debug)]
struct Foo {
    bar: Vec<u8>,
}

fn main() {}
