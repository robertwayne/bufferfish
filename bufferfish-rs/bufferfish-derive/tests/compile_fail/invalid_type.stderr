error: type can not be serialized into a bufferfish
 --> tests/compile_fail/invalid_type.rs:5:10
  |
5 |     bar: Vec<u8>,
  |          ^^^^^^^
